woocommerce-gateway-ebanx/services/class-wc-ebanx-constants.php

```diff
@@ -139,6 +139,20 @@
 	);
 
 	/**
+	 * The cash payments processed by EBANX
+	 *
+	 * @var array
+	 */
+	public static $CASH_PAYMENTS = array(
```

Comment:
They are not cash payments. A cash payment is a payment, not a string. They are cash payment gateways code.